src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/EditMetadataByCreatingNewTypeAttribute.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This needs to be added to the ref assembly as well.  Presumably we want to expose it from https://github.com/dotnet/runtime/blob/main/src/libraries/System.Runtime.Loader/ref/System.Runtime.Loader.cs ?